### ISSUE TITLE
Sometimes samples start playing but immediately stop

### ISSUE THREAD

**Issue by villermen** [team-thyme/soundboard#47]
On both mobile and desktop, and happens arbitrarily. Once in every like 10 tries or so. Does not happen to samples that have already been played, even the once that had this bug occur to. So second click will always play, regardless of whether it errored before. Makes me think it is a howler bug (on 2.0.0-beta14), also happened on beta13.

**Comment by villermen** [comment]
Might be related to #48.

**Comment by villermen** [comment]
```
602howler.min.js:2 Uncaught TypeError: Failed to set the 'buffer' property on 'AudioBufferSourceNode': The provided value is not of type 'AudioBuffer'.i @ howler.min.js:2
```

**Comment by villermen** [comment]
Seems to happen mainly on longer samples.

**Comment by villermen** [comment]
onStop event is emitted when sample is loaded (why though?), which clears the sample from the list of samples that are supposed to be playing. For bigger samples this can lead into the sample cutting off after it has begun playing (not in the latest commit, as I've updated a check for that). Now instead it won't even start playing. Turns out that's also not desired.

**Comment by villermen** [comment]
Fixed in 2d45465a35d60a90ec4732590a6c048f5f57b2ff.
